apps/docs/pages/guides/auth/social-login/auth-linkedin.mdx

```diff
@@ -10,6 +10,12 @@ To enable LinkedIn Auth for your project, you need to set up a LinkedIn OAuth ap
 
 ## Overview
 
+<Admonition type="caution" label="update">
+
+Linkedin has changed some of their APIs for OAuth which won't work with the current Supabase implementation. See [here](https://learn.microsoft.com/en-us/linkedin/shared/authentication/authorization-code-flow?context=linkedin%2Fcontext&tabs=HTTPS1) for more information.
```

Comment:
```suggestion
LinkedIn has updated their OAuth APIs, which has broken the implementation for the current Supabase implementation. See [here](https://learn.microsoft.com/en-us/linkedin/shared/authentication/authorization-code-flow?context=linkedin%2Fcontext&tabs=HTTPS1) for more information.
```

Reply:
we also need to share what we are doing about this. Will we deprecate it? Are we looking into it? How can developers track our progress?